Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -96,13 +100,25 @@ public function nearestContentCollection(Node $node, string $nodePath): Node
     }
 
     /**
-     * Generate a label for a node with a chaining mechanism. To be used in nodetype definitions.
+     * Generate a label for a node with a chaining mechanism. To be used in NodeType definition:
+     *
+     *     'Vendor.Site:MyContent':
+     *       label: "${Neos.Node.labelForNode(node).prefix('foo')}"
+     *
      */
     public function labelForNode(Node $node): NodeLabelToken
     {
         return new NodeLabelToken($node);
     }
 
+    /**
+     * Renders the actual node label based on the NodeType definition in Fusion.
+     */
+    public function renderLabel(Node $node): string
```

Comment:
do we need this when we have https://github.com/neos/neos-development-collection/pull/5046 's 

`${q(node).label()}`

?